husky_directory/app.py

```diff
@@ -100,7 +100,11 @@ def dispatch_request(self):
         request_input = self.injector.get(SearchDirectoryInput)
         self.logger.info(f"searching for {request_input}")
         request_output = self.directory.search_directory(request_input)
-        return jsonify(request_output.dict(by_alias=True))
+        result = {
```

Comment:
What does the result object contain after line 105 runs?

Reply:
```
{
   "query description x": {"people": [ {person...}, {person...}, {person...}]},
   "query description y": {"people": [ {person...}, {person...}, {person...}]},
   ...
}
```

Reply:
![Screen Shot 2021-01-08 at 9 33 29 AM](https://user-images.githubusercontent.com/1092941/104054409-e1836000-51a1-11eb-8297-65ae4a08237e.png)

Reply:
(for now; I'll probably change this into a more explicit model once I have the other searches working and feel like the underlying models are solidified)

Reply:
Ah, so 105 returns a `descr` and `result` for each "item" in `request_output`?

Reply:
I'm not second guessing your code, just upping my understanding 👍

Reply:
yes, this is known as "dictionary comprehension." 

the end result is the same as if i had written:

```
results = {}
for description, result in request_output.items():
    results[description] = result.dict(by_alias=True)
```

Dictionary comprehensions (and list/set comprehensions) are efficient built-in python constructs.